pages/common/git-abort.md

```diff
@@ -0,0 +1,9 @@
+# git abort
+
+> Abort a ongoing rebase, merge or cherry-pick.
```

Comment:
```suggestion
> Abort an ongoing rebase, merge, or cherry-pick.
```

* It should be "an" instead of "a".
* Added an [oxford comma](https://en.wikipedia.org/wiki/Serial_comma).

Reply:
Yes, good spot.

the an was forgot when changing the `ongoing` word